docs/tutorial/overview.md

```diff
@@ -68,13 +65,31 @@ A NOMAD user account is required if you want to upload, share, publish, or analy
 
 Creating a NOMAD user account is quick and free. Follow these steps to set up your account (click on the arrows to navigate between steps):
 
-<div class="image-slider">
+<div class="image-slider" id="slider1">
```

Comment:
These are very nice! I did not see though at first that they were carousels. Perhaps you can indicate in the text or even directly above/below it like a caption?